### ISSUE TITLE
Plantar plate injury

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11186]
**Preferred term label:**
Plantar plate injury

**Synonyms**

**Definition (free text, please give PubMed ID)**
Foot MRI or ultrasound showing edema or partial vs complete tear of the plantar plate of the toe, with or without associated metatarsophalangeal (MTP) joint synovitis

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sesamoid injuries of the hallux

**Your nano-attribution (ORCID)**
